src/TrafficSign/TrafficSignHelper.ts

```diff
@@ -0,0 +1,347 @@
+  diversionR,
+  endOfLimitations,
+  exitHighway,
+  exitInnerCityRoad,
+  exitLimitedAccessRoad,
+  fileUnknown,
+  giveWay,
+  highway,
+  highwayEnd,
+  icy,
+  limitedAccessRoad,
+  limitedAccessRoadEnd,
+  livingStreet,
+  livingStreetEnd,
+  minimumSpeed30,
+  minimumSpeedEnd30,
+  narrowRoad,
+  narrowRoadL,
+  narrowRoadR,
+  noEntry,
+  noOvertaking,
+  noOvertakingEnd,
+  noOvertakingTrucks,
+  noOvertakingTrucksEnd,
+  noTraffic,
+  oneWayLeft,
+  oneWayRight,
+  oneWayStraight,
+  parkingGarage,
+  parkingLot,
+  passL,
+  passR,
+  pedestrianCrossing,
+  pedestrians,
+  priorityRoad,
+  priorityRoadEnd,
+  rightOfWay,
+  roadWorks,
+  roundabout,
+  sCurveL,
+  sCurveR,
+  slipperyRoad,
+  speedLimit10,
+  speedLimit100,
+  speedLimit110,
+  speedLimit120,
+  speedLimit130,
+  speedLimit20,
+  speedLimit30,
+  speedLimit40,
+  speedLimit5,
+  speedLimit50,
+  speedLimit60,
+  speedLimit70,
+  speedLimit80,
+  speedLimit90,
+  speedLimitEnd10,
+  speedLimitEnd100,
+  speedLimitEnd110,
+  speedLimitEnd120,
+  speedLimitEnd130,
+  speedLimitEnd20,
+  speedLimitEnd30,
+  speedLimitEnd40,
+  speedLimitEnd5,
+  speedLimitEnd50,
+  speedLimitEnd60,
+  speedLimitEnd70,
+  speedLimitEnd80,
+  speedLimitEnd90,
+  speedLimitZone30,
+  speedLimitZoneEnd30,
+  steepAscent10,
+  steepAscent11,
+  steepAscent12,
+  steepAscent13,
+  steepAscent14,
+  steepAscent15,
+  steepAscent16,
+  steepAscent17,
+  steepAscent18,
+  steepAscent19,
+  steepAscent20,
+  steepAscent21,
+  steepAscent22,
+  steepAscent23,
+  steepAscent24,
+  steepAscent25,
+  steepAscent4,
+  steepAscent5,
+  steepAscent6,
+  steepAscent7,
+  steepAscent8,
+  steepAscent9,
+  steepDescent10,
+  steepDescent11,
+  steepDescent12,
+  steepDescent13,
+  steepDescent14,
+  steepDescent15,
+  steepDescent16,
+  steepDescent17,
+  steepDescent18,
+  steepDescent19,
+  steepDescent20,
+  steepDescent21,
+  steepDescent22,
+  steepDescent23,
+  steepDescent24,
+  steepDescent25,
+  steepDescent4,
+  steepDescent5,
+  steepDescent6,
+  steepDescent7,
+  steepDescent8,
+  steepDescent9,
+  stop,
+  straightOrLeft,
+  straightOrRight,
+  trafficLight,
+  trafficQueues,
+  tunnel,
+  turnAheadL,
+  turnAheadR,
+  turnL,
+  turnR,
+  unevenRoad,
+  urbanArea,
+  urbanAreaEnd,
+  warningPedestrianCrossing,
+  weightLimit,
+  widthLimit
+};
+
+/**
+ * For a given sign type and value from the json road file, the function will return the imported instance of the sign. If not an unknownFile will be imported.
+ * @param {string} type The type of sign.
+ * @param {string} value The value of the sign.
+ * @returns {string} The imported image string.
+ */
+const getRoadSignName = (type: string, value: string) => {
+  const signsWithValues = [
+    "speedLimit",
+    "speedLimitEnd",
+    "steepAscent",
+    "steepDescent",
+    "minimumSpeed",
+    "minimumSpeedEnd",
+    "speedLimitZone",
+    "speedLimitZoneEnd"
+  ];
+  if (signsWithValues.some(signWithValue => type === signWithValue)) {
```

Comment:
I find this option easier to read and I think is actually what you want to check rather than some

```suggestion
  if (signsWithValues.includes(type)) {
```

Reply:
There's also no need for the else here. We generally try to avoid if/else where the if returns or errors.

```
if (condition) {
  return something
}

return somethingElse
```